iron-fit-behavior.html

```diff
@@ -171,14 +171,14 @@
      * the memoized data.
      */
     resetFit: function() {
-      if (!this._fitInfo || !this._fitInfo.sizedBy.height) {
-        this.sizingTarget.style.maxHeight = '';
-        this.style.top = this._fitInfo ? this._fitInfo.inlineStyle.top : '';
-      }
       if (!this._fitInfo || !this._fitInfo.sizedBy.width) {
         this.sizingTarget.style.maxWidth = '';
-        this.style.left = this._fitInfo ? this._fitInfo.inlineStyle.left : '';
       }
+      if (!this._fitInfo || !this._fitInfo.sizedBy.height) {
+        this.sizingTarget.style.maxHeight = '';
+      }
+      this.style.top = this._fitInfo ? this._fitInfo.inlineStyle.top : '';
+      this.style.left = this._fitInfo ? this._fitInfo.inlineStyle.left : '';
```

Comment:
Here we are resetting the memoized data.
`this._fitInfo.inlineStyle` will contain the original `top, left`, and is independent from `this._fitInfo.sizedBy` which should be used for resetting `maxWidth, maxHeight`.
By associating them together, what happens is the following:
- on the first `refit()`, `left, top` are not set, hence we can calculate the proper position and set those values (memoized data)
- on the second `refit()`, `left, top` will not be reset, so the positioning will stay as is, with the current `top, left` (which is the memoized data, not the original `top, left`

I believe this is a safe change.